vdj_ann/src/annotate.rs

```diff
@@ -30,6 +30,16 @@ use vector_utils::{
     reverse_sort, unique_sort, VecUtils,
 };
 
+// Replacing ann object with format: { ( start on sequence, match length, ref tig, start on ref tig, mismatches on sequence ) }.
+#[derive(Clone, Copy, Eq, PartialEq, Ord, PartialOrd)]
+pub struct Annotation {
+    pub seq_start: i32,
+    pub match_len: i32,
+    pub ref_tig: i32,
+    pub ref_start: i32,
+    pub mismatches: i32,
```

Comment:
Since this is a widely-used type, please add a docstring for the struct as well as for each field.